src/Compilers/VisualBasic/Test/IOperation/IOperation/IOperationTests_IForEachLoopStatement.vb

```diff
@@ -1111,6 +1109,96 @@ IForEachLoopOperation (LoopKind.ForEach, Continue Label Id: 0, Exit Label Id: 1)
             VerifyOperationTreeAndDiagnosticsForTest(Of ForEachBlockSyntax)(source, expectedOperationTree, expectedDiagnostics)
         End Sub
 
+        <CompilerTrait(CompilerFeature.IOperation)>
+        <Fact(), WorkItem(17602, "https://github.com/dotnet/roslyn/issues/17602")>
+        Public Sub IForEachLoopStatement_CastArrayToIEnumerable()
```

Comment:
No equivalent in C#?

Reply:
Ah, nevermind, it already existed.

---
In reply to: [198597880](https://github.com/dotnet/roslyn/pull/28158#discussion_r198597880) [](ancestors = 198597880)